web_src/css/modules/normalize.css

```diff
@@ -25,7 +25,7 @@ Use a better box model (opinionated).
 }
 
 html {
-  line-height: 1.15; /* 1. Correct the line height in all browsers. */
+  line-height: 1.2; /* 1. Correct the line height in all browsers. The original "1.15" is too small for some fonts. "1.2" is roughly the same as default "normal" */
```

Comment:
If there is a `normal`, why don't we just use that?

Reply:
`normal` seems better yes, gives the browser some leeway to adjust.

Reply:
Then let's try using "normal" bb26380bc13f149380c01993d95b8c79204700f3